modular_zubbers/modules/modular_weapons/code/company_and_or_faction_based/carwo_defense_systems/ammo/carwo.dm

```diff
@@ -0,0 +1,20 @@
+/obj/projectile/bullet/c35sol
+	damage = 20
+	wound_bonus = -10
+	wound_falloff_tile = -10
+
+/obj/projectile/bullet/c40sol
+	damage = 30
+	wound_bonus = -5
+	wound_falloff_tile = 0
+
+/obj/projectile/bullet/c40sol/pierce
+	damage = 25
+	bare_wound_bonus = -15
+	armour_penetration = 20
+
+/obj/projectile/bullet/c40sol/incendiary
+	damage = 18
+
+/obj/projectile/bullet/c35sol/ripper
+	damage = 5
```

Comment:
5 damage?? that's crazy low

Reply:
The wounding chance is very high and you can combine multiple ammo types into a magazine

Reply:
ok im sorry but in their right mind is gonna combine different type of ammo into a magazine???? that's not a valid argument